django_tables2_column_shifter/templates/django_tables2_column_shifter/bootstrap2.html

```diff
@@ -27,7 +27,7 @@
                     <span class="caret"></span>
                 </button>
                 <ul class="dropdown-menu" style="min-width:350px; padding:5px; cursor:pointer;">
-                    {% for column in table.columns %}
+                    {% for column in table.hideable_columns %}
```

Comment:
The loop should always iterate over all columns.
Of all the columns, some are active and some are not.

Pseudo code:
```
{% for col in all_cols%}
         {% if column in table.hideable_columns%}
                ## Button is active (old functionality)
         {% else%}
                ## Position is inactive (gray and not clicable) 
```

For `in active` elements you can add extra css (like 'not_hideable`) class have easy access from JS or CSS to remove for example this elements if you need.